frontend/server/src/Controllers/Contest.php

```diff
@@ -2671,6 +2675,20 @@ public static function apiCreate(\OmegaUp\Request $r) {
             ['partial','all_or_nothing','max_per_group'],
         );
         $checkPlagiarism = $r->ensureOptionalBool('check_plagiarism') ?? false;
+        
+        // Handle recommended flag - only available for admins and support team
+        $recommendedValue = false;
+        $recommended = $r->ensureOptionalBool('recommended');
+        if (!is_null($recommended)) {
+            if (!\OmegaUp\Authorization::isSystemAdmin($r->identity) && 
+                !\OmegaUp\Authorization::isSupportTeamMember($r->identity)) {
```

Comment:
Ok, to avoid nesting conditions, you could just use:
```suggestion
        if (
            !is_null($recommended) &&
            !\OmegaUp\Authorization::isSupportTeamMember($r->identity)
        ) {
```
?

Reply:
Here 
if ( !is_null($recommended) && 
    !\OmegaUp\Authorization::isSystemAdmin($r->identity) && 
    !\OmegaUp\Authorization::isSupportTeamMember($r->identity )
    ) {
    ..
    }
    I have changed to this SO it would not throw exception for SystemAdmin too .
    is it okay??